### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by pmartin2308**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,9 +51,9 @@ In all projects there needs to be a reliable method of source/version control. [
 ![Branching Strategy Overview](./assets/branching.png)
 
 **Branches:**
-- `master`/`release`: This is the live or production version of the code. The single source of truth for the most up-to-date code.
-- `dev`/`staging`: This is the staging version of the code. Testing of new features should be done here prior to a release to `master`/`release` branch.
-- `feature`: This branch should be where development takes place. Branch can be for individual developers or issues/bugs/features. Sometimes if the feature is big there will have to be subbranches merging into this. Naming isn't as important as long as the `pull request` to `dev`/`staging` includes the appropriate ticket number.
+- `master`/`release`/`prod`: This is the live or production version of the code. The single source of truth for the most up-to-date code.
+- `dev`/`staging`/`full`/`partial`: This is the staging version of the code. Testing of new features should be done here prior to a release to `master`/`release`/`prod` branch.
```
Comment (jefflombard) [inline README.md]: What do you think about changing this to just `dev`/`staging`/`full`? Just so we differentiate between a `staging` environment and a `development` environment.

File: README.md [hunk 2]
```diff
@@ -54,10 +62,19 @@ In all projects there needs to be a reliable method of source/version control. [
...
+Source Code Management is typically required for larger heavily customized implementations of Salesforce. Its benefits are as follows. 
+Maintains historical record of changes for code review, rollback of changes and management of packages to be released. 
+Facilitates means for continuous integration across packages and sandboxes.
+Facilitates release management by using Metadata Packages.  
+Salesforce customizations can be accessed as text or XML through a number of interfaces. The most common are the Metadata API and the SFDX commandline tool which uses the Metadata API.
+
 One of the following should be used:
 - GitHub
 - BitBucket
+- Subversion
```
Comment (jefflombard) [inline README.md]: Do we use subversion, VSTS, or TFS, on any *non-legacy* projects? If so why not git? Is this something customers would request?

File: README.md [hunk 3]
```diff
@@ -54,10 +62,19 @@ In all projects there needs to be a reliable method of source/version control. [
 ### Salesforce Specific Source Control
 Salesforce often uses Production as the source of truth, which can be problematic. In which case see the `Source Code Management` section of [Software Development in Salesforce](https://www.linkedin.com/pulse/software-development-salesforce-tuan-lu/).
 
-## Git Repo Hosting
+## Source Control Tools
+Source Code Management is typically required for larger heavily customized implementations of Salesforce. Its benefits are as follows. 
+Maintains historical record of changes for code review, rollback of changes and management of packages to be released. 
+Facilitates means for continuous integration across packages and sandboxes.
+Facilitates release management by using Metadata Packages.  
+Salesforce customizations can be accessed as text or XML through a number of interfaces. The most common are the Metadata API and the SFDX commandline tool which uses the Metadata API.
```
Comment (jefflombard) [inline README.md:70]: Let's move line `66` - `70` under Versioning at line `47` seems to make more sense?

File: README.md [hunk 4]
```diff
@@ -117,7 +134,7 @@ Some of the editors our devs use include:
 - [VS Code](https://code.visualstudio.com/Vim)
 
 # Testing
-Testing Strategies should be determined on a per project basis.
+Testing Strategies should be determined on a per project basis.
```
Comment (jefflombard) [inline README.md:136]: Remove whitespace eol

**Review by jefflombard — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pmartin2308) [reply README.md]: We have used subversion, VSTS, and TFS.  VSTS and TFS might come up more and more with VS Code as that is what a lot of .Net use for Source Control.  I don't think it hurts to leave them there as we are just showing examples of what clients may use.  Maybe we could add a *preferred next to our preferred Source Control, IDE, etc.?

File: README.md (hunk 1 above)
Comment (pmartin2308) [reply README.md]: I'd think Staging is probably partial or full and Dev is probably partial or dev

File: README.md (hunk 3 above)
Comment (pmartin2308) [reply README.md:70]: This should be fine.

File: README.md (hunk 1 above)
Comment (jefflombard) [reply README.md]: Okay, I'll reword to make this more clear

**Review by jefflombard — APPROVED** [review]

**Pull request merged**